graphblas_algorithms/interface.py

```diff
@@ -83,6 +83,14 @@ def convert(graph, weight=None):
             return Graph.from_networkx(graph, weight=weight)
         raise TypeError(f"Unsupported type of graph: {type(graph)}")
 
+    @staticmethod
+    def convert_to_nx(obj, *, name=None):
+        from .classes import Graph
+
+        if isinstance(obj, Graph):
+            obj = obj.to_networkx()
```

Comment:
Can we check `isinstance(obj, Matrix)` and convert it to a scipy sparse array if scipy is installed?

We can also merge this PR and update things later.